Lib9c/Battle/Simulator.cs

```diff
@@ -68,7 +68,7 @@ MaterialItemSheet materialItemSheet
             {
                 try
                 {
-                    var data = itemSelector.SelectV2(1).First();
```

Comment:
is this result of v2 and v3 are same?

Reply:
If the sum of the `item_ratio` parameters in the `StageSheet` was not 1, the result of v2 and v3 would have been different. However, the sum of the option ratio parameters in the `StageSheet` is 1.